nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -77,11 +77,8 @@ def test_plot_anat():
     import pylab as pl
     pl.switch_backend('template')
     img = _generate_img()
-    # Test saving with empty plot
-    z_slicer = plot_anat(anat_img=False, display_mode='z')
-    with tempfile.TemporaryFile(suffix='.png') as fp:
-        z_slicer.savefig(fp.name)
 
+    # Test saving with empty plot
```

Comment:
Why?

Updated the rest.


2015-03-27 15:25 GMT+01:00 Loïc Estève <notifications@github.com>:

> In nilearn/plotting/tests/test_img_plotting.py
> <https://github.com/nilearn/nilearn/pull/520#discussion_r27298367>:
>
> >
> > +    # Test saving with empty plot
>
> This comment is lying
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/520/files#r27298367>.
>



-- 
Viele Grüße,
Danilo